### ISSUE TITLE
Cosmetics for Players with Cracked Account

### ISSUE THREAD

**Issue by XSprizen** [Solar-Tweaks/Solar-Tweaks#1147]
**Describe the bug**
Right now the cosmetics obtained by the solar socket don't work if you're playing with a cracked account.
They do work on singleplayer but not on multiplayer (any public server).
This includes Lunar Emotes as well as Cosmetics.
Cosmetics also are visible in the avatar but aren't visible if you log onto any server.

**Expected behavior**
The Lunar Cosmetics and Emotes should be visible when playing multiplayer (should be visible on servers)
just like if your playing on a real bought Minecraft account.

**Actual behaviour**
The Lunar Cosmetics and Emotes are only visible in avatar and singleplayer. but not when you log onto multiplayer.

**Comment by XSprizen** [comment]
also along with the cosmetics and emotes, the skin changer also doesn't work.
would be really awesome if bot of these problems were fixed :D

**Comment by Herohaider999** [comment]
yeah...

**Comment by RichardDorian** [comment]
> also along with the cosmetics and emotes, the skin changer also doesn't work. would be really awesome if both of these problems were fixed :D

Please move this to another issue.

**Comment by RichardDorian** [comment]
Making cosmetics and emotes work on public cracked server is a very hard task. Lunar Client uses UUID as a unique identifier **per player**. When using cracked accounts you don't have this unique identifier which means making it compatible with Lunar Client is hard, but we managed to do it anyway. When connecting to a public crack server, the distant server assigns another UUID to the player. Therefore, the client believes the player's UUID is a one completely different from the one used with Solar Socket.

We didn't know about this when we made cracked accounts possible in Solar Socket. Since it is a problem many of you faced, we will work on it for Solar Socket V2 (which is under work right now).
